.github/workflows/build-test.yml

```
@@ -0,0 +1,28 @@
+name: NodeJS build
+
+on:
```

yamllint (1.35.1)

[warning] truthy value should be one of [false, true]

(truthy)